### BIG differences, this is diff of wrong files.
### CLUS "System Class LIST"
### and CLHS topic selection, http://www.lispworks.com/documentation/lw50/CLHS/Body/a_list.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:list.txt	2017-04-27 22:01:05.000000000 +0100
+++ saved-pages-clus/cl:types:list.txt	2017-04-27 22:01:06.000000000 +0100
@@ -1,4 +1,36 @@
-LIST
-Please select which reference to LIST you intended
+System Class LIST

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+list, sequence, t
+Description
+A list is a chain of conses in which the car of each cons is an element of the list, and the cdr of each cons is either the next link in the chain or a terminating atom.
+ A proper list is a chain of conses terminated by the empty list, '(), which is itself a proper list.
+ A dotted list is a list which has a terminating atom that is not the empty list.
+ A circular list is a chain of conses that has no termination because some cons in the chain is the cdr of a later cons.
+Dotted lists and circular lists are also lists, but usually the unqualified term "list" within this specification means proper list. Nevertheless, the type list unambiguously includes dotted lists and circular lists.
+For each element of a list there is a cons. The empty list has no elements and is not a cons.
+The types cons and null form an exhaustive partition of the type list.
+See Also
+{\secref\LeftParen}, {\secref\PrintingListsAndConses}
\ No newline at end of file
